### ISSUE TITLE
Grey square in place of image

### ISSUE THREAD

**Issue by angeloborrelli** [navjotdhanawat/dynamic-html-pdf#20]
Hello,
Plugin has always worked in last 1 year and never got a problem producing PDF with images with `<img>` tag.
But today we found that new pdf's are not displaying header logo anymore. See attached file on top-left you will see a grey little square instead of image that is a classic PNG you can check/view/download at [https://b2b.o-jersey.com/uploads/file-1633732451913.png](https://b2b.o-jersey.com/uploads/file-1633732451913.png)
Everything is still working pretty well in development (windows) but for some reason we cannot understand, it stopped working on linux. We tried with JPG from same domain/path and problem remains. We tried with another random image from another domain and it works! This strange unexpected behaviour seems to be bound to server (dynamic-html-pdf 1.0.4 + Ubuntu 18.04 LTS + nginx 1.14.0 + nodejs 13.13). Can you check and help us to solve this issue? Let me know if you need more info/details.
Thank you in advance.
[order#3.pdf](https://github.com/navjotdhanawat/dynamic-html-pdf/files/7314484/order.3.pdf)

### UPDATE:
We also tried to produce a PDF with same image (https://b2b.o-jersey.com/uploads/file-1633732451913.png) from windows dev environment and it works perfectly.

### WORKAROUND:
We successfully made an experiment by putting inside `<img>` tag a simple ENDPOINT of another server. Here below php code snippet:
```
$url = $_GET["image"];
$headers = get_headers( $url );
$type = $headers[3];
$size = $headers[4];
$imageString = file_get_contents($url);
header('Content-Type: '.$type);
header('Content-length: '.$size);
echo $imageString;
exit();
```
Basically it does nothing but read the remote image (see `image` parameter) and "spit it out" as its own. So we can use it as `<img src="https://www.../path/to/endpoint.php?image=https://b2b.o-jersey.com/uploads/file-1633732451913.png">` and it works (see attached order#3 (FIXED).pdf). Don't know if it can help anyone but that's how we could solve this issue...still wondering why it fails to display when `src` is set to original image url.
[order#3 (FIXED).pdf](https://github.com/navjotdhanawat/dynamic-html-pdf/files/7315606/order.3.FIXED.pdf)

**Comment by JanScheeleDev** [comment]
We are having the same problem, but I cannot confirm the server setup. Dev version is working fine, live version is not. I checked the dependencies of this lib and found similar issues here: https://github.com/marcbachmann/node-html-pdf/issues/652. Maybe it's an SSL issue with phantomJS. The Let's Crypt root certificate was invalid by end of september, which would fit the date your problems occured.
